torch/testing/_internal/distributed/_sharded_tensor/__init__.py

```diff
@@ -75,6 +80,91 @@ def assert_sharded_tensor_equal(self, st1, st2):
         self.assertEqual(st1.sharding_spec(), st2.sharding_spec())
         self.assertEqual(len(st1.remote_shards()), len(st2.remote_shards()))
 
+class BinaryCmpOpTestBase(ShardedTensorTestBase):
```

Comment:
nit: This can just be in `binary_cmp.py` since it is used only in one file?

Reply:
good point.